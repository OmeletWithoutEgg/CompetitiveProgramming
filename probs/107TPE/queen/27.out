139550.36 29958041.15
