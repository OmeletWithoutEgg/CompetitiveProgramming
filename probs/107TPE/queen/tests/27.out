-11685.91 9420716.42
